app/src/components/slidesContainer.js

```diff
@@ -55,8 +55,16 @@ export class SlidesContainer extends Component {
   }
 
   set activeSlide(value) {
-    this.slides.forEach((slide) => slide.classList.remove("active"));
-    this.slides[value].classList.add("active");
+    const currentSlide = this.slides[value];
+    this.slides.forEach((slide) => {
+      slide.classList.remove("active");
+      slide.setAttribute("inert", true);
+      slide.setAttribute("aria-hidden", true);
+    });
+    currentSlide.classList.add("active");
+    currentSlide.removeAttribute("inert");
+    currentSlide.setAttribute("aria-hidden", false);
+    currentSlide.focus();
```

Comment:
might make the code more readable to do this within the `forEach` loop within an `if / else` block.